runtime/compiler/control/J9Options.cpp

```diff
@@ -3146,7 +3145,7 @@ bool J9::Options::feLatePostProcess(void * base, TR::OptionSet * optionSet)
       }
 #if defined(J9VM_OPT_CRIU_SUPPORT)
    else if (fsdStatus == FSDInitStatus::FSDInit_NotInitialized
-            && javaVM->internalVMFunctions->isDebugOnRestoreEnabled(vmThread))
```

Comment:
This was the last reference to `vmThread`; please remove the declaration on line 3112.

Reply:
Removed the declaration.
Also removed a similar declaration at `HookedByTheJit.cpp` L7238.